test/cpp/naming/generate_resolver_component_tests.bzl

```diff
@@ -33,6 +33,7 @@ def generate_resolver_component_tests():
             "//:gpr",
             "//test/cpp/util:test_config",
         ],
+        tags = ["exclude_windows"],
```

Comment:
I'd be keen to enable these resolver tests to run Windows at least eventually :)... (so far, one has needed to build and run these tests on Windows manually).

It make sense to file a bug to enable these tests to run on Windows as a follow-up to this PR?

Reply:
They heavily rely on POSIX. Each target need to go through significant changes on thread handling, network stack, etc. to be able to run on Windows with MSVC. I'm not anticipating that to be done any time soon.

Reply:
I mean, feel free to create issues to list what'd be left to do for Windows, yes :-) But don't flag it as P0 or P1.

Reply:
Actually, unless the tests have changed recently, we should be able to build and run `address_sorting_test*` and `resolver_component_test*` on Windows (I added Windows support to those tests a while back for manual runs). Is it possible to remove the `exclude_windows` tags from those targets?

I definitely agree that the `resolver_component_tests_runner_invoker*` targets need significant refactoring to run on Windows though - I'll track that as something to do later.

Reply:
edit: for  `resolver_component_test*`, we can't "run" this target out of the box, but it should compile on Windows

Reply:
@apolcyn address_sorting_test requires sys/types.h